doit/dependency.py

```diff
@@ -528,7 +528,7 @@ def status_is_ignore(self, task):
         return self._get(task.name, "ignore:")
 
     # TODO add option to log this
-    def get_status(self, task, tasks_dict):
+    def get_status(self, task, tasks_dict, rebuild_log=None):
```

Comment:
that is so ugly. `rebuild_log` is a return value not a parameter. This is not a public API. I make sure doit has 100% code coverage to give confidence in making this kind of changes.